src/Microsoft.AspNetCore.Blazor.Browser.JS/Microsoft.AspNetCore.Blazor.Browser.JS.csproj

```diff
@@ -5,18 +5,20 @@
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <TypeScriptToolsVersion>Latest</TypeScriptToolsVersion>
     <DefaultItemExcludes>${DefaultItemExcludes};node_modules\**</DefaultItemExcludes>
+    <IsPackable>false</IsPackable>
```

Comment:
Since the beginning, we've been shipping `Microsoft.AspNetCore.Blazor.Browser.JS` to NuGet.org, but nothing actually depends on it. It only exists to run as part of our build in this repo. As far as I can tell, we shouldn't be shipping this as a package to NuGet.

Reply:
Sounds good.